.github/actions/helm-docs/entrypoint.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# ENVS FROM ACTION YAML 
+PERSONAL_TOKEN="$INPUT_PERSONAL_TOKEN"
+SRC_PATH="$INPUT_SRC_PATH"
+COMMIT_MESSAGE="$INPUT_COMMIT_MESSAGE"
+USERNAME="$INPUT_USERNAME"
+EMAIL="$INPUT_EMAIL"
+TEMPLATE_FILE="$INPUT_TEMPLATE_FILE"
+IGNORED_DIRS="$INPUT_IGNORED_DIRS"
+GIT_PUSH="$INPUT_GIT_PUSH"
+# OTHER ENVS
+BASE_PATH=$(pwd)
+
+
+echo "-----Information:---------------------------"
+git --version
+helm-docs --version
+echo "Base path is: $BASE_PATH"
+if [ "${TEMPLATE_FILE}" != "" ]
+then
+    echo "Templates file location is: $TEMPLATE_FILE"
+else
+    echo "Template file is not provided"
+fi
+
+echo "----.helmdocsignore Status:----------------"
+
+FILE=$(pwd)/.helmdocsignore
+if test -f "$FILE"; then
+    echo "$FILE exists."
+else 
+    echo "Creating .helmdocsignore file"
+    touch .helmdocsignore
```

Comment:
What happens with the file in the end? Is it also checked in?

Reply:
at the moment it gets created and also gets checked in 
If preferred I can change it, that .helmdocsignore will not be pushed (does sound cleaner):

HELMDOCSIGNORE_EXISTED=""
FILE=$BASE_PATH/.helmdocsignore
if [test -f "$FILE"]
then 
   echo "$FILE exists."
   HELMDOCSIGNORE_EXISTED="true"
else 
    touch .helmdocsignore
    HELMDOCSIGNORE_EXISTED="false"
fi

....
(before commiting and pushing:)
if [ "HELMDOCSIGNORE_EXISTED" == "false"]
then 
    rm -f $BASE_PATH/.helmdocsignore
fi

Reply:
Let us discuss this with in the team